ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/router-bgp.j2

```diff
@@ -21,6 +21,24 @@
 {%         for bgp_default in router_bgp.bgp_defaults | arista.avd.default([]) %}
 | {{ bgp_default }} |
 {%         endfor %}
+{%         if router_bgp.graceful_restart.enabled is arista.avd.defined(true) and router_bgp.graceful_restart.restart_time is arista.avd.defined %}
+| graceful-restart restart-time {{ router_bgp.graceful_restart.restart_time }} |
+{%         endif %}
+{%         if router_bgp.graceful_restart.stalepath_time is arista.avd.defined %}
+| graceful-restart stalepath-time {{ router_bgp.graceful_restart.stalepath_time }} |
```

Comment:
Why is `stalepath-time` printed without `enabled:true`?
Maybe we should just have an outer if for `enabled(true)`

Reply:
I copied the exact logic of the router-bgp EOS template.

Reply:
I checked on EOS. There is no difference between `restart-time` and `stalepath-time` that could explain the different logic. I expect it is an artifact of historic behavior.
I would gate all of them behind the `enabled: true` flag.